.github/workflows/ci.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         if: ${{ matrix.composer-deps == 'latest' }}
       - name: Code style check
         run: vendor/bin/php-cs-fixer fix --dry-run
+        if: ${{ matrix.php-versions == '8.1' }}
```

Comment:
php-cs-fixer can't run on PHP 8.2 in the current version.
If i update the cs-fixer you will get some other fixes in the code and don't have PHP 7.4 support anymore.

But the cs-fix should be enough in one PHP-Versions and shouldn't run on every PHP Version